packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -86,6 +115,38 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
     return result;
   }
 
+  /// Stores the engine layer created for this layer in order to reuse engine
+  /// resources across frames for better app performance.
+  ///
+  /// Set this field to the value returned by one of [ui.SceneBuilder]'s "push"
+  /// methods, e.g. [ui.SceneBuilder.pushOpacity].
```

Comment:
move this paragraph to the docs for the setter

Reply:
Done.